kubetest/main.go

```diff
@@ -495,7 +492,7 @@ func migrateGcpEnvAndOptions(o *options) error {
 			name:   "--gcp-network",
 		},
 		{
-			env:    nodeImage,
+			env:    "KUBE_NODE_OS_DISTRIBUTION",
 			option: &o.gcpNodeImage,
```

Comment:
I believe it leaves them alone? Let's head over and consult the unit test :grin: